testgrid/config/config.yaml

```diff
@@ -474,6 +478,8 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-alpha-features-release-1-6
 - name: ci-kubernetes-e2e-gci-gce-ingress-release-1-6
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-ingress-release-1-6
+  alert_stale_results_hours: 24
+  num_failures_to_alert: 5
```

Comment:
Why is ingress CI different (using 5 instead of 3)?

Reply:
typo, will revert to default